site/_apps/stencil-ionic-hn.txt

```diff
@@ -7,7 +7,7 @@ libraries:
       _ Ionic/Core
 service-worker: 'Application Shell'
 server-side-rendering: 'No'
-api: 'node-hnapi'
+api: 'HNPWA api'
```

Comment:
@davideast FYI that this is now using the HNPWA API as you suggested

Reply:
\o/